src/init.js

```diff
@@ -0,0 +1,346 @@
+import { isObject } from 'lodash'
+import { authActions, queryActions, storageActions } from './actions'
+
+export default function init (firebase, config, dispatch) {
```

Comment:
It would be nice to see this as a `const` set to a function instead of a named function to limit scope unless there is a reason it is being named:

```js
export default (firebase, config, dispatch) => {
```